tests/app/helpers.py

```diff
@@ -71,9 +51,16 @@ def setup_authorization(self):
     def do_not_provide_access_token(self):
         self.app.wsgi_app = self.app.wsgi_app.app
 
-    def apply_migrations(self):
+    def setup_frameworks(self):
         with self.app.app_context():
-            upgrade(self.config, 'head')
+            if Framework.query.count() == 0:
```

Comment:
Is it possible to exclude reference tables from the delete data phase? We could keep some record of bootstrapped tables and ignore them.

I can see some of these tables becoming more complex in our future.